.lintstagedrc.cjs

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  "!(*.ts|*.tsx)": "prettier --write --ignore-unknown",
+  "*.{ts,tsx}": ["eslint --fix", "prettier --write"],
+  "*{package.json,package-lock.json}": () => ["syncpack list-mismatches"],
```

Comment:
moving to a `js` file allows running files that don't take filenames

Reply:
Ah, nice!